pkg/connector/client.go

```diff
@@ -0,0 +1,95 @@
+package connector
+
+import (
+	"context"
+	"strconv"
+
+	"go.mau.fi/mautrix-meta/messagix"
+	//"go.mau.fi/mautrix-meta/pkg/store"
+	"maunium.net/go/mautrix/bridgev2"
+	"maunium.net/go/mautrix/bridgev2/networkid"
+)
+
+type MetaClient struct {
+	Main *MetaConnector
+	//session *store.MetaSession
```

Comment:
Should we store the Session just for easier database updates? Could make the Session wrap Client instead of Cookies... might be better anyway, hmm...